.isort.cfg

```diff
@@ -0,0 +1,11 @@
+[settings]
+force_alphabetical_sort_within_sections=true
+multi_line_output=3
+force_grid_wrap=2
+include_trailing_comma=true
+src_paths=lib
+known_first_party=galaxy_test
+no_lines_before=LOCALFOLDER
+reverse_relative=true
+skip_gitignore=true
+line_length=200
```

Comment:
```suggestion
extend_skip=doc/source/conf.py,lib/galaxy/util/jstree.py
force_alphabetical_sort_within_sections=true
force_grid_wrap=2  # Override value from profile=black, but black is still happy
line_length=120  # Same as for black
no_lines_before=LOCALFOLDER
profile=black
reverse_relative=true
skip_gitignore=true
skip_glob=database/*  # Make isort run faster
src_paths=lib
```

Reply:
When I applied this to #11162 I had to drop the inline comments from the .cfg.  Otherwise isort blew up.